contrib/forms/clinical_notes/new.php

```diff
@@ -87,32 +87,25 @@ function cbcell($name, $desc, $colname)
  // If updating an existing form...
  //
     if ($formid) {
-        $query = "UPDATE form_clinical_notes SET "      .
-         "history = '"          . $_POST['form_history']     . "', " .
-         "examination = '"      . $_POST['form_examination'] . "', " .
-         "plan = '"             . $_POST['form_plan']        . "', " .
-         "followup_required = " . rbvalue('fu_required')     . ", "  .
-         "followup_timing = '$fu_timing'"                    . " "   .
-      // "outcome = "           . rbvalue('outcome')         . ", "  .
-      // "destination = "       . rbvalue('destination')     . " "   .
-         "WHERE id = '$formid'";
-        sqlStatement($query);
+        $query = "UPDATE form_clinical_notes SET
+         history = ?, 
+         examination = ?,      
+         plan = ?,           
+         followup_required = ?,
+         followup_timing = ?,                  
+      // outcome = ?     rbvalue('outcome')        
+      // destination = ?   rbvalue('destination')    
+         WHERE id = ?";
+         
+        sqlStatement($query, array($_POST['form_history'], $_POST['form_examination'], $_POST['form_plan'], rbvalue('fu_required'), $fu_timing, $formid));
```

Comment:
Lets fixup the rbvalue function to work with binding. In that function:
Change:
```
return "'$tmp'";
```
to:
```
return "$tmp";
```
(note I just removed the single quotes)